src/main/resources/templates/db_scripts/views/receiving_history_view.sql

```diff
@@ -51,5 +51,10 @@ SELECT pieces.id AS id,
            'receivingStatus', pieces.jsonb ->>'receivingStatus',
            'supplement', pieces.jsonb ->>'supplement',
            'title', po_line.jsonb ->>'titleOrPackage')::jsonb AS metadata
-FROM pieces LEFT OUTER JOIN po_line ON pieces.jsonb ->>'poLineId' = po_line.jsonb->>'id' LEFT OUTER JOIN purchase_order
-    ON po_line.jsonb->>'purchaseOrderId' = purchase_order.jsonb->>'id';
+FROM pieces
+  LEFT OUTER JOIN po_line
+    ON pieces.jsonb ->>'poLineId' = po_line.jsonb->>'id'
```

Comment:
let's use columns: id and poLineId instead of jsonb representations, this approach will definetly use indexes (rmb automatically creates indexes for foreign keys)

Reply:
Fixed, by the way the script execution  will be triggered in Q because of this change made in previous PR:
https://github.com/folio-org/mod-orders-storage/pull/379/files#diff-4db9b6a16c40e8569cf1f57856808c27dff44495be574ae83559974077c193e4R12
Because of this change we did not have changes in schema.json in scope of this particular PR